src/pages/CreateEvent.jsx

```diff
@@ -6,8 +6,8 @@ import { yupResolver } from '@hookform/resolvers/yup';
 import { Form, Button } from 'antd';
 import moment from 'moment';
 import uploadWaiver from '../components/events/utils';
-import EventsGeneralInfo from '../components/events/CreateEvent/EventsGeneralInfo';
-import EventsAdditionalInfo from '../components/events/CreateEvent/EventsAdditionalInfo';
+import EventsGeneralInfo from '../components/events/createEvent/EventsGeneralInfo';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../components/events/createEvent/EventsGeneralInfo"